products/keda.md

```diff
@@ -0,0 +1,72 @@
+---
+title: KEDA
+category: server-app
+tags: kubernetes
```

Comment:
```suggestion
```


`kubernetes` is not an existing tag, not sure we want to add it. @captn3m0 do we want to add this tag ? What are the other products that could use it in such case ?

@nunoadrego just to explain, we want to have a very limited set of meaningful tags, so new one must be added carefully.

Reply:
Other projects that can use this tag:
- Kubernetes
- Azure Kubernetes Service
- Amazon EKS
- Google Kubernetes Engine
- etcd (?)
- Istio

But as @captn3m0 said, a CNCF tag might be another possibility.

Reply:
For k8s, we could perhaps use a separate `kubernetes-distribution` tag. (That would just apply to k8s, aks, eks, gke pages). 

cncf here should be good enough.